docs/_includes/projects/docs/compose/my-app.js

```diff
@@ -0,0 +1,51 @@
+import { LitElement, html } from '@polymer/lit-element';
+
+class MyApp extends LitElement {  
+  static get properties(){
+    return {
+      headerText: { type: String },
```

Comment:
These properties don't need to use an object to specify the type of the property. You can just say `property: Type`

```suggestion
      headerText: String,
```

Reply:
No, LitElement requires an object with `type` field for proper attribute-to-property deserialization (not so important for `String`, but necessary for other types). See README.

Reply:
Oh, my bad, @keanulee, will delete